cnvlib/_version.py

```diff
@@ -1 +1 @@
-__version__ = "0.7.1.dev0"
+__version__ = "0.7.2.dev0"
```

Comment:
I don't know of that on. I always use a "pre next version" tag. I use alpha (0.7.2a0) but I think dev works fine too.